### PR TITLE
DEVX-2255: Create documentation

### PR DESCRIPTION
**Pull request by AmyJZhao**
## Description

Created documentation for Template Registry Github repo, validation checks, and API

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by roribio — APPROVED** [review]

File: src/pages/guides/api/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+# Template Registry API
+
+## Authentication
+
+A valid Adobe IMS access token is required for `POST` and `DELETE` actions. 
+
+## Authorization
+
+- Admin: is a member of an Adobe IMS organization listed in the `ADMIN_IMS_ORGANIZATIONS` environment varialble
```
Comment (slitviachenko) [inline src/pages/guides/api/index.md:15]: A small typo.
```suggestion
- Admin: is a member of an Adobe IMS organization listed in the `ADMIN_IMS_ORGANIZATIONS` environment variable.
```

File: src/pages/guides/github/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Only the original owner that added the template to the registry may update a template. If not the original owner, contact the Developer Experience team at devx-services@adobe.com to update a template.
+
+## Remove a template
+
+##### Request to remove an entry from the Template Registry. 
+
+1. Navigate to [issues](https://github.com/adobe/aio-template-submission/issues) in the Template Registry submission Github repository.
+2. Click `New issue` to create a new issue.
+3. Select `Get started` for `Template Removal Request`.
+4. Fill in the name of the NPM package that contains the template.Include scope (e.g. @adobe/app-builder-template).
```
Comment (slitviachenko) [inline src/pages/guides/github/index.md:49]:
```suggestion
4. Fill in the name of the NPM package that contains the template. Include scope (e.g. @adobe/app-builder-template).
```

File: src/pages/index.md [hunk 3]
```diff
@@ -1,27 +1,60 @@
...
+
+[Template Validation Library](guides/template_validation_library)
+
+Run validation checks on App Builder templates for verification and endorsement purposes. 
+
+<DiscoverBlock slots="heading, link, text"/>
+
+### API Reference
+
+Try the [Adobe App Builder Template Registry API](api/)   with Swagger UI. Explore, make calls, with full endpoint descriptions.
```
Comment (slitviachenko) [inline src/pages/index.md:53]:
```suggestion
Try the [Adobe App Builder Template Registry API](api/) with Swagger UI. Explore, make calls, with full endpoint descriptions.
```

**Review by slitviachenko — CHANGES_REQUESTED** [review]
@AmyJZhao , I recently changed `/api/` to `/apis/` as OpenWhisk hardcodes it in a path.
Could you please replace all `/api/v1/` to `/apis/v1/`?

**Review by slitviachenko — CHANGES_REQUESTED** [review]
@AmyJZhao , could you please also update https://github.com/AdobeDocs/app-builder-template-registry/blob/main/src/openapi.yaml to be in sync with https://github.com/magento-eqp/aio-template-registry-api/blob/main/openapi.yaml ? I recently updated it a little bit.

**Review by slitviachenko — APPROVED** [review]

**Pull request merged**